osh/hub/scripts/osh-retention

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Copyright contributors to the OpenScanHub project.
+
+"""
+Script for cron that performs the retention policy
+"""
+import datetime
+import logging
+import os
+import shutil
+from datetime import date
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'osh.hub.settings'
+
+
+def deleteScansResults(scansToDelete):
+    from kobo.hub import models
+    for scan in scansToDelete:
+        try:
+            pathToDelete = models.Task.get_task_dir(scan.id)
+            if os.path.isdir(pathToDelete):
+                shutil.rmtree(pathToDelete)
```

Comment:
Please make the directory removal unconditional because right now it introduces a TOCTOU bug (https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use).  If the target path does not exist, it will be caught by the `except` statement anyway.